config/routes.rb

```diff
@@ -1,3 +1,21 @@
 Rails.application.routes.draw do
+
+  get 'auth/:provider/callback', to: 'sessions#create'
+  get 'auth/failure', to: redirect('/')
+  get 'signout', to: 'sessions#destroy', as: 'signout'
+
+  resources :sessions, only: [:create, :destroy]
+  resources :home, only: [:show]
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
```

Comment:
remove this comment (maybe save it somewhere else if you need to reference it?) and the comment on line 12.